plugins/plugin-github/che-plugin-github-ext-github/src/main/java/org/eclipse/che/ide/ext/github/server/rest/GitHubService.java

```diff
@@ -264,6 +275,21 @@ public GitHubPullRequest createPullRequest(@PathParam("user") String user, @Path
         return repoList;
     }
 
```

Comment:
Different repos belong to different organizations. To display in the same way as Dashboard does.